lambda/translcore.ml

```diff
@@ -893,7 +893,7 @@ and transl_apply ~scopes
       ?(mode=Alloc_heap)
       lam sargs loc
   =
-  let lapply funct args loc pos =
+  let lapply funct args loc pos mode =
```

Comment:
I know that this predates the PR, but I notice that this `mode` parameter is used in all cases except one: the one that extends an `Lapply`. Is that deliberate or an oversight caused by using `with` syntax?

Reply:
Very well spotted. This was a bug, fix applied and testcase now in `regions.ml`.